docs/openAPI.json

```diff
@@ -9007,10 +9014,6 @@
                     }
                 }
             },
-            "return_format": {
```

Comment:
I would recommend keeping this in the description anyways, might be helpful to have an elaboration of the possible values